### ISSUE TITLE
default value for subroutine arguments

### ISSUE THREAD

**Issue by deg026** [Camelcade/Perl5-IDEA#1757]
can we add the dafault value persing for arguments ?
i usually do it in that way:

```
sub getPackagePrice
{
    my $SID = shift;
    my $period = shift || 12;
    my $years = shift || 1;
```

can we have support for this variant, and to use it in autocompilation system

**Comment by hurricup** [comment]
Do you mean that plugin does not see `$period` and `$years` as parameters?

**Comment by deg026** [comment]
yes, this too, when there is not only "shift;" system didnnt understand it as parameter
but also i mean to show default values of parameters as on attached image:
![parameters](https://user-images.githubusercontent.com/38157104/38541562-be5ebfa8-3c9f-11e8-8420-d0612cf4f796.png)

**Comment by hurricup** [comment]
Oh, I see. Looks like a double request :)
